daemon/set.go

```diff
@@ -0,0 +1,53 @@
+package daemon
+
+import (
+	"github.com/docker/docker/runconfig"
+)
+
+func (daemon *Daemon) ContainerSet(name string, hostConfig *runconfig.HostConfig) ([]string, error) {
+	warnings, err := daemon.verifyHostConfig(hostConfig)
+	if err != nil {
+		return warnings, err
+	}
+
+	container, err := daemon.Get(name)
```

Comment:
Its a minor thing, but I'd prefer to check the name before the hostConfig data because to me that's the most important piece of information we get from the client so let's check that first.